validator.go

```diff
@@ -229,9 +254,8 @@ func (mv *Validator) newTypeValidator(t reflect.Type) ValidationFunc {
 				state.popPath()
 			}
 		}
-	default:
-		return okValidation
 	}
+	return postTagValidation(valFn)
```

Comment:
This custom validation after the tag validation didn't exist in another form yet, right? (The existing custom validation relies on custom tags if I'm not mistaken)
In that case it could be nice to adapt the README to mention this new functionality as well, WDYT?